css/main.css

```diff
@@ -161,15 +186,24 @@ a.logo{
 }
 .navbar-gridzilla a{
     border-right: 1px solid #ececec;
-    padding: 0px 15px !important;
-    margin: 10px 0 !important;
+    padding: 0px 23px !important;
+    margin: 17px 0 !important;
```

Comment:
Please, do not use `!important`!
http://www.quizful.net/post/specificity-of-css-selectors